pkgs/applications/audio/mpg123/default.nix

```diff
@@ -18,7 +21,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
-  buildInputs = [ perl ] ++ lib.optional (!stdenv.isDarwin) alsaLib;
+  buildInputs = [ perl ] ++ lib.optional (!stdenv.isDarwin) alsaLib
```

Comment:
```suggestion
  buildInputs = [ perl ] ++ lib.optionals !stdenv.isDarwin [ alsaLib ]
```

Reply:
Seems pointless to me… I think the mpg123 derivation has other problems that I'd rather address:

> ```
>   Modules ................. enabled
>   Checked audio modules ... alsa tinyalsa oss coreaudio sndio sun win32 win32_wasapi os2 esd jack portaudio pulse sdl nas arts openal dummy
>   Detected audio support .. alsa oss dummy
>   Default output module ... alsa
> ```

I'll prolly drop the mpg123 changes and just make a new PR for bump & refactoring, it pulls in alot of rebuilds that I really don't care about in this PR and sidplay builds fine with out123's dummy driver.

Reply:
mpg123 changes removed.